src/extensions/default/Autosave/main.js

```diff
@@ -50,6 +51,11 @@ define(function (require, exports, module) {
             doc.releaseRef();
             clearTimeout(pending[path]);
             delete pending[path];
+
+            // if statement cheks if there is no more pending saves
+            if(Object.keys(pending).length === 0) {
+                BrambleEvents.triggerProjectSaved(path);
```

Comment:
We don't pass along a path when the project is saved (e.g., it doesn't make sense, since you're saying "the entire project, all files, are saved.").  Remove `path` here.